### PR TITLE
Update changes file for PR #10082

### PR DESCRIPTION
**Pull request by feerrenrut**
### Link to issue number:
Changes file update for #10082

### Summary of the issue:
#10082 introduced a large change, introducing a GUI for Vision Enhancement Providers. The change log requires several updates to match.

### CODE REVIEW

**Review by LeonarddeR — APPROVED** [review]
Looks good to me.

File: user_docs/en/changes.t2t [hunk 1]
```diff
@@ -23,6 +22,12 @@ What's New in NVDA
  - Figures are are also reported. They are considered objects and therefore navigable with the o quick navigation key.
 - In Internet Explorer, Google Chrome and Mozilla Firefox, article elements are now reported with object navigation, and optionally in browse mode if turned on in Document Formatting settings. (#10424)
 - Support for out-of-process iframes in Mozilla Firefox. (#9672)
+- Added screen curtain, which when enabled, makes the whole screen black on Windows 8 and later. (#7857)
+ - Added a script to enable screen curtain (until next restart with one press, or always while NVDA is runninging with two presses), no default gesture is assigned.
```
Comment (lukaszgo1) [inline user_docs/en/changes.t2t]: runninging  -> running

File: user_docs/en/changes.t2t [hunk 2]
```diff
@@ -23,6 +22,12 @@ What's New in NVDA
  - Figures are are also reported. They are considered objects and therefore navigable with the o quick navigation key.
```
Comment (XLTechie) [inline user_docs/en/changes.t2t]:
```suggestion
Figures are also reported. They are considered objects and therefore navigable with the o quick navigation key.
```

File: user_docs/en/changes.t2t [hunk 3]
```diff
@@ -23,6 +22,12 @@ What's New in NVDA
  - Figures are are also reported. They are considered objects and therefore navigable with the o quick navigation key.
 - In Internet Explorer, Google Chrome and Mozilla Firefox, article elements are now reported with object navigation, and optionally in browse mode if turned on in Document Formatting settings. (#10424)
 - Support for out-of-process iframes in Mozilla Firefox. (#9672)
+- Added screen curtain, which when enabled, makes the whole screen black on Windows 8 and later. (#7857)
+ - Added a script to enable screen curtain (until next restart with one press, or always while NVDA is runninging with two presses), no default gesture is assigned.
+ - Can be enabled and configured via the added 'vision' category in NVDA's settings dialog.
+- Added screen highlight functionality to NVDA. (#971, #9064)
+ - Highlighting of the focus, navigator object, and browse mode caret position can be enabled and configured via the added 'vision' category in NVDA's settings dialog.
```
Comment (XLTechie) [inline user_docs/en/changes.t2t]:
```suggestion
 - Highlighting of the focus, navigator object, and browse mode caret position can be enabled and configured via the new 'vision' category in NVDA's settings dialog.
```

**Comment by feerrenrut** [comment]
Thanks @XLTechie and @lukaszgo1, these changes were added in 90ae5d6

**Pull request merged**